### DISCUSSION TITLE
Feature Request: UML Diagrams

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/outline#1472] [General] [answered]
Render UML diagrams using [Mermaid](https://mermaidjs.github.io/). For example, this will produce a sequence diagram:

```mermaid
sequenceDiagram
Alice ->> Bob: Hello Bob, how are you?
Bob-->>John: How about you John?
Bob--x Alice: I am good thanks!
Bob-x John: I am good thanks!
Note right of John: Bob thinks a long<br/>long time, so long<br/>that the text does<br/>not fit on a row.

Bob-->Alice: Checking with John...
Alice->John: Yes... John, how are you?
```
![image](https://user-images.githubusercontent.com/23284276/90821582-8394cf80-e309-11ea-8204-111cae6b55e4.png)

**Comment by damageboy** [comment, marked as answer]
FYI, This is a dup of:
https://github.com/outline/outline/discussions/1364
